### PR TITLE
Updates the JATS 1.2 XML document with additional information

### LINKED ISSUE #3557: JATS XML Stub is missing information
**Is your feature request related to a problem? Please describe.**
The following things have been identified as missing from our JATS stub after looking into it:

1. ORCID
2. CI statement
3. Correspondence Author
4. History, eg Recieved and Accepted dates
5. Issue TItles
6. Funding information

**Describe the solution you'd like**
Add these to the JATS stub

### PR DESCRIPTION
**Pull request by ajrbyers**
- Adds additional fields to the JATS XML inline with recommended practice
- Fixes bug that meant issue titles would not be in JATS
- Closes #3557

### CODE REVIEW

File: src/templates/common/encoding/article_jats_1_2.xml [hunk 1]
```diff
@@ -53,16 +56,54 @@
...
+                </date>
+                {% endif %}
+            </history>
+            {% endif %}
             <volume seq="{{ article.issue.order }}">{{ article.issue.volume }}</volume>
             <issue>{{ article.issue.issue }}</issue>
             <issue-id>{{ article.issue.pk }}</issue-id>
             {% if article.issue.title %}
-            <issue-title xml:lang="en">{{ article.issue.title }}</issue-id>
+            <issue-title xml:lang="en">{{ article.issue.title }}</issue-title>
```
Comment (mauromsl) [inline src/templates/common/encoding/article_jats_1_2.xml:106]: there is no `article.issue.title`, did you mean `article.issue.issue_title` or `article.issue.display_title`?

File: src/templates/common/encoding/article_jats_1_2.xml [hunk 2]
```diff
@@ -81,6 +122,18 @@
                 {% endfor %}
             </kwd-group>
             {% endif %}
+
+            {% if article.funders.all %}
+                {% for funder in article.funders.all %}
+                    <funding-group>
+                        <award-group>
+                            <funding-source id="{{ funder.fundref_id }}">{{ funder.name }}
```
Comment (mauromsl) [inline src/templates/common/encoding/article_jats_1_2.xml:130]: `funder.fundref_id` can be null, which would violate jats constrains over `@id` attribute

**Review by mauromsl — CHANGES_REQUESTED** [review]
A couple of comments inline

File: src/templates/common/encoding/article_jats_1_2.xml (hunk 1 above)
Comment (ajrbyers) [reply src/templates/common/encoding/article_jats_1_2.xml:106]: My change here is to `</issue-title>` not to the content of the tag. Has this been wrong the whole time?

**Review by mauromsl — APPROVED** [review]

**Comment by ajrbyers** [comment]
Pushing an update to fix tests momentarily.

**Review by joemull — APPROVED** [review]

**Pull request merged**